azure-functions/acbs-function/api.js

```diff
@@ -112,47 +134,54 @@ const patchToAcbs = async (apiRef, acbsInput, eTag) => {
 
     return axios({
       method: 'patch',
-      url: `${acbs}/${apiRef}`,
-      auth: {
-        username,
-        password,
-      },
+      url,
       headers: {
-        'Content-Type': 'application/json',
+        ...headers,
         ...additionalHeader,
       },
-      data: acbsInput,
-    }).catch((e) => {
-      console.error('Error calling PATCH to ACBS');
+      data: payload,
+    }).catch((error) => {
+      console.error('Error calling ACBS PATCH /%s', endpoint);
+
       return {
-        status: e.response ? e.response.status : e,
-        data: { error: e.response ? e.response.data : e },
+        status: error.response ? error.response.status : error,
+        data: { error: error.response ? error.response.data : error },
       };
     });
   }
-  return {
-    status: 400,
-    data: {},
-  };
+
+  return badRequest;
 };
 
-const getFacility = (facilityId) => getACBS(`facility/${facilityId}`);
-const getLoanId = (facilityId) => getACBS(`facility/${facilityId}/loan`);
-const createParty = (acbsInput) => postToAcbs('party', acbsInput);
-const createDeal = (acbsInput) => postToAcbs('deal', acbsInput);
-const createDealInvestor = (acbsInput) => postToAcbs('deal/investor', acbsInput);
-const createDealGuarantee = (acbsInput) => postToAcbs('deal/guarantee', acbsInput);
-const createFacility = (acbsInput) => postToAcbs('facility', acbsInput);
-const createFacilityInvestor = (acbsInput) => postToAcbs('facility/investor', acbsInput);
-const createFacilityCovenantId = (acbsInput) => postToAcbs('numbers', acbsInput);
-const createFacilityCovenant = (acbsInput) => postToAcbs('facility/covenant', acbsInput);
-const createFacilityGuarantee = (acbsInput) => postToAcbs('facility/guarantee', acbsInput);
-const createCodeValueTransaction = (acbsInput) => postToAcbs('facility/codeValueTransaction', acbsInput);
-const createFacilityLoan = (acbsInput) => postToAcbs('facility/loan', acbsInput);
-const createFacilityFee = (acbsInput) => postToAcbs('facility/fixedFee', acbsInput);
-const updateFacility = (facilityId, updateType, acbsInput, etag) => putToAcbs(`facility/${facilityId}?op=${updateType}`, acbsInput, etag);
-const updateFacilityLoan = (facilityId, loanId, acbsInput) => patchToAcbs(`facility/${facilityId}/loan/${loanId}`, acbsInput);
-const updateFacilityLoanAmount = (facilityId, loanId, acbsInput) => postToAcbs(`facility/${facilityId}/loan/${loanId}/amountAmendment`, acbsInput);
+/**
+ * Various ACBS API calls, consuming myriads of
```

Comment:
Could this change to just `// ACBS API calls`? I'm not sure the rest of the description adds anything.

Reply:
I think it is good to have descriptive comments have shortened it though.